src/app/guide/pages/data-submission-guide/data-submission-guide.component.scss

```diff
@@ -0,0 +1,45 @@
+.guide {
+  margin-top: 5rem;
+}
+
+ul.side-navigation {
+  list-style: none;
+  margin: 0 4rem 0 0;
+  padding: 0;
+
+  li.side-navigation__item {
+    padding: .8rem  .7rem;
+    font-weight: 400;
+    text-decoration: none;
+    border-bottom: 1px #D0D0CE solid;
+
+    &.active {
+      background: #D1E3F6;
```

Comment:
Do we have a central location where we set stylistic attributes like fonts, colors, etc.?